### PR TITLE
Added new fields for Jenkins Total and Busy executors

### PR DESCRIPTION
**Pull request by vikkyomkar**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

**Comment by vikkyomkar** [comment]
Fixes #6843

**Comment by vikkyomkar** [comment]
@danielnelson can you please review this PR ?

**Comment by vikkyomkar** [comment]
@danielnelson Can I get some eyes on it ?

File: plugins/inputs/jenkins/jenkins.go [hunk 1]
```diff
@@ -244,6 +245,17 @@ func (j *Jenkins) gatherNodesData(acc telegraf.Accumulator) {
 		acc.AddError(err)
 		return
 	}
+
+	// get total and busy executors
+	if nodeResp.TotalExecutors != 0 {
```
Comment (danielnelson) [inline plugins/inputs/jenkins/jenkins.go]: I think we should add this measurement even when total executors is 0.

Comment (vikkyomkar) [reply plugins/inputs/jenkins/jenkins.go]: Updated code as per the suggestion

**Pull request merged**